src/resources/MachineLearning/FilterConditions.ts

```diff
@@ -0,0 +1,22 @@
+export enum Operator {
+    Equals = 'EQUALS',
+    NotEquals = 'NOT_EQUALS',
+}
+
+export interface FilterConditions {
+    /**
+     * The name of the field.
+     * Example: status
+     */
+    field: string;
+    /**
+     * The operator to use to evaluate the condition.
+     * Must be EQUALS or NOT_EQUALS
+     */
+    operator: Operator;
```

Comment:
Suggestion: The enum is superfluous.

```suggestion
    operator: 'EQUALS' | 'NOT_EQUALS';
```